content/en/api/versions.md

```diff
@@ -0,0 +1,47 @@
+---
+title: API Versions
+description: How to detect new features within the Mastodon API.
+menu:
+  docs:
+    weight: 40
+    parent: api
+---
+
+Since Mastodon version 4.3.0, the [Instance Details]({{< relref "methods/instance#v2" >}}) endpoint has returned a property indicating the Mastodon API Version, which is separate from the release number, and is just an integer that can be used to detect the presence of new methods or properties on the Mastodon API.
+
+```json
+// GET /api/v2/instance
+{
+  // ...
+  "api_versions": {
+    "mastodon": 5,
+  },
+  // ...
+}
+```
+
+## Changelog
+### Version 1
+
+- *None*
+
+### Version 2
+
+- Grouped notifications API: https://github.com/mastodon/mastodon/pull/31840
+
+### Version 3
+
+- API for updating attribution domains: https://github.com/mastodon/mastodon/pull/32730
+
+### Version 4
+
+- Media deletion methods: https://github.com/mastodon/mastodon/pull/34035
```

Comment:
https://github.com/mastodon/mastodon/pull/33988 is documented in `content/en/methods/statuses.md`, https://github.com/mastodon/mastodon/pull/33991 is undocumented. Can you document the latter?